util/helm/helm_test.go

```diff
@@ -219,6 +219,29 @@ func TestAPIVersions(t *testing.T) {
 	assert.Equal(t, "sample/v2", objs[0].GetAPIVersion())
 }
 
+func TestIssue22606(t *testing.T) {
```

Comment:
```suggestion
func TestIssue22606(t *testing.T) {
```

please add a better name for this test.

Reply:
Agree with @nitishfy. Maybe add a more descriptive name for the function and add the link to the issue as a comment.

Reply:
Thanks you both, I renamed it.